py2pack/__init__.py

```diff
@@ -97,9 +97,9 @@ def _get_template_dirs():
     is important. The first found template from the first found dir wins!"""
     return filter(lambda x: os.path.exists(x), [
         # user dir
-        os.path.join(os.path.expanduser('~'), '.py2pack', 'templates'),
```

Comment:
Why? Yes, I don’t expect support for Windows anytime soon, but defensive programming is always better.

Also, wouldn’t this be a nice opportunity to move from `~/.py2pack` to XDG-compliant `~/.config/py2pack`?

Reply:
yes, you right

Reply:
yes, i did